### DISCUSSION TITLE
Cookie Consent Support

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [expo/expo#31450] [Polls]
So while we already know, that cookie consent is required on web it is also required in native apps.
Therefore an implementation would be nice to easily ask the user for permission of different cookie types.

**Comment by ydrea** [comment]
f*** me, but it's legally required (in the EU at least...)

**Reply by NilsBaumgartner1994** [reply]
Sadly yes. Therefore I think a nice module for that would be nice
